opencensus/metrics_quickstart/main.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// metrics_quickstart is an example of exporting a custom metric from
+// OpenCensus to Stackdriver.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"golang.org/x/exp/rand"
+)
+
+var (
+	// The latency in milliseconds.
+	latencyMs = stats.Float64("demo/latency", "The latency in milliseconds", "ms")
+
+	latencyView = &view.View{
+		Name:        "demo/latency",
+		Measure:     latencyMs,
+		Description: "The distribution of the latencies",
+
+		// Latency in buckets:
+		// [>=0ms, >=100ms, >=200ms, >=400ms, >=1s, >=2s, >=4s]
+		Aggregation: view.Distribution(0, 100, 200, 400, 1000, 2000, 4000),
+	}
+)
+
+var projectID = os.Getenv("GOOGLE_CLOUD_PROJECT")
+
+func main() {
+	if projectID == "" {
+		log.Fatal("The GOOGLE_CLOUD_PROJECT environment variable must be set.")
```

Comment:
stackdriver.NewExporter resolves the project ID from the ADC, this env var is not required.